var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -226,8 +240,9 @@ def install(self, spec, prefix):
         else:
             options.append('--with-clanguage=C')
 
-        # Help PETSc pick up Scalapack from MKL:
-        if 'scalapack' in spec:
+        # Help PETSc pick up Scalapack from MKL
+        # 'satisfies' is used to only pick up scalapack when needed
```

Comment:
could you please add a note here that `+mumps+mpi~int64` is the pre-condition to use `scalapack`.